docs/api/tools/block-explorer/batches-view.md

```diff
@@ -0,0 +1,29 @@
+# Browse batches
+
+Batches are like groups of transactions that are processed together in one [block](../../../dev/developer-guides/transactions/blocks.md#blocks) before they're added to the main blockchain. It increases the transaction throughput of zkSync Era and reduces its transaction fees.
+
+By putting several transactions into a single batch, we reduce the number of transactions that need to be processed and validated, this results in faster transaction processing times and lower fees for users interacting with zkSync Era.
+
+![Browse batches!](../../../assets/images/batches.png "View batches on zkSync")
```

Comment:
This should be of the Block Explorer home screen, or dashboard. This picture here is for the next section.